docs/begin.md

```diff
@@ -9,12 +9,27 @@ sidebar_label: Introduction
+Internally, we train these models on large databases of episodes, but externally we provide support for running BlueWhale inside [OpenAI Gym](gym.openai.com).  BlueWhale is extremely fast and can train models with thousands of parameters on billions of rows of data.  All of our models export to the caffe2 predictor after training so the trained model can be efficiently served on hundreds of thousands of machines to billions of people.
 
-Advances in RL theory, including the advent of Deep Query Networks and Deep Actor-Critic models, allow us to use function approximation to approach problems with large state and action spaces.  This project, called BlueWhale, contains Deep RL implementations built on [PyTorch](http://pytorch.org/) and [caffe2](http://caffe2.ai/). Internally, we train these models on large databases of episodes, but externally we provide support for running BlueWhale inside [OpenAI Gym](gym.openai.com).  BlueWhale is extremely fast and can train models with 10M+ parameters on billions of rows of data.
+## RL + Recommender Systems
+
+Most machine learning models in industry are *supervised*: they predict a value given a set of features (also called inputs) and labels (also called the "ground-truth").  These models can inform us *what* will be the label given a set of features, but they don't tell us *how* to maximize some utility.  We must define a *policy*, which takes in the outputs of these supervised models and decides which items to recommend.
+
+While part of the utility is immediate (e.g. a click or a thumbs-up on a post), there are other facets of the utility that can only be measured over time such as long-term engagement.  RL allows us to maximize these long-term benefits and balance them with the short term benefits effectively.
+
+## RL + Infrastructure
+
+Many problems in infrastructure involve stateful, dynamical systems.  For example, one may want to stream video to a cell phone from a server in the cloud.  The available bandwidth and the amount of the video that has already been buffered can be encoded into a state, and the action can be the quality of the next video segment to request.  As low-quality buffers fill up or the bandwidth increases, we can shift to requesting higher quality video and improve the user experience.
+
+
+## RL + You
+
+While historically, RL has been primarily used in the context of robotics and game-playing, it can be employed in a variety of problem spaces. At Facebook, we're working on using RL at scale: suggesting people you may know, notifying you about fiend & page updates, optimizing our streaming video bitrate, and more.  If you use BlueWhale for your project, [let us know](contact.html)!
```

Comment:
suggestion:

"If you use BlueWhale for your project or have any questions about how you can use BlueWhale"